### ISSUE TITLE
Debian packaging

### ISSUE THREAD

**Issue by stephanlachnit** [jackun/USBqemu-wheel#64]
I'm interested in packaging this for Debian, and I see that you already did some packaging.

I would have to change a couple of things before uploading, the question is whether you want me to maintain it here upstream (I can do that, already do it for Lutris and Minihook kinda)? I would use Ubuntu 20.04 as base.

**Comment by jackun** [comment]
Dunno which version of PCSX2 sticks around in 20.04 but all (I think) plugins are (getting) merged into 1.7

**Comment by stephanlachnit** [comment]
Ah cool, so it won't be necessary to have this as a separate package around? Then I think the effort is not really worth it.

**Comment by jackun** [comment]
Yeah probably. I'll probably keep updating separate plugin for some time. But if some fixes are necessary for deb packaging then feel free to PR it.
